scripts/scil_visualize_bundles.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Visualize bundles.
+
+Example usages:
+
+# Visualize streamlines as tubes, each bundle with a different color
+$ scil_visualize_bundles path_to_bundles/ --shape tube --random_coloring 1337
+
+# Visualize a tractogram with each streamlines drawn as lines, colored with
+# their local orientation, but only load 1 in 10 streamlines
+$ scil_visualize_bundles tractogram.trk --shape line --subsample 10
+
+# Visualize CSTs as large tubes and color them from a list of colors in a file
+$ scil_visualize_bundles path_to_bundles/CST_* --width 0.5 \
+    --color_list colors.txt
+"""
+
+import argparse
+import colorsys
+import glob
+import itertools
+import nibabel as nib
+import numpy as np
+import os
+import random
+
+from dipy.tracking.streamline import set_number_of_points
+from fury import window, actor
+
+from scilpy.io.utils import assert_inputs_exist, parser_color_type
+
+
+streamline_actor = {'tube': actor.streamtube,
+                    'line': actor.line}
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__, formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_bundles', nargs='+',
+                   help='List of tractography files supported by nibabel.')
+    coloring_group = p.add_mutually_exclusive_group()
+    coloring_group.add_argument('--random_coloring', metavar='SEED', type=int,
+                                help='Assign a random color to bundles.')
+    coloring_group.add_argument('--color_list', type=str, metavar='FILE',
+                                help='File containing colors for each bundle.'
+                                'Color values must be separated by a space'
+                                ' and each color must be on its own line.'
+                                'Can include opacity.')
+    p.add_argument('--shape', type=str,
+                   choices=['line', 'tube'], default='tube',
+                   help='Display streamlines either as lines or tubes.')
+    p.add_argument('--width', type=float, default=0.05,
+                   help='Width of tubes or lines representing streamlines')
+    p.add_argument('--subsample', metavar='N', type=int, default=1,
+                   help='Only load 1 in N streamlines.')
+    p.add_argument('--downsample', metavar='N', type=int, default=None,
```

Comment:
Please add an option to use coloring from scalars_per_point or properties_per_streamline. This would be very nice.